src/components/utils/hooks/useXmtpClient.tsx

```diff
@@ -47,7 +47,11 @@ const useXmtpClient = (cacheOnly = false) => {
           storeKeys(await signer.getAddress(), keys);
         }
 
-        const xmtp = await Client.create(null, { env: XMTP_ENV, privateKeyOverride: keys });
+        const xmtp = await Client.create(null, {
```

Comment:
You probably want the appVersion on the getKeys line above as well. That actually does do an API request to download the key bundle, so we should track it along with the rest.